hoot-hadoop/src/main/cpp/hoot/hadoop/paint-nodes/PaintNodesMapper.h

```diff
@@ -46,8 +46,7 @@ class PaintNodesMapper : public OsmMapMapper
   Envelope _envelope;
   double _pixelSize;
   int _width, _height;
-
-  HadoopPipes::MapContext* _context;
+  bool _initialized;
```

Comment:
The parent class actually has a `HadoopPipes::MaxContext* _context` member already, this is completely unnecessary and confusing.